helm/aqua-app-enforcer/values.yaml

```
@@ -34,18 +34,18 @@ securityContext:
   #     - SYSLOG
   #     - SYS_CHROOT
 
-hostRunPath:  # pks - /var/vcap/sys/run/docker
+hostRunPath: # pks - /var/vcap/sys/run/docker
 
 rbac:
   enabled: true
   roleRef:
 
 gate:
-  host: aqua-gateway-svc  # for envoy deployment use host for aqua-lb and port 443
+  host: aqua-gateway-svc # for envoy deployment use host for aqua-lb and port 443
   port: 8443
 
 image:
-  repository: giantswarm/aqua-enforcer
+  repository: enforcer
```

We need to retain this
```suggestion
  repository: giantswarm/aqua-enforcer
```

---

damn it... I did those :facepalm: